proc-macros/src/lib.rs

```diff
@@ -293,23 +293,21 @@ pub(crate) mod visitor;
 ///         // The stream API can be used to pipe items from the underlying stream
 ///         // as subscription responses.
 ///         async fn sub_override_notif_method(&self, pending: PendingSubscriptionSink) -> SubscriptionResult {
-///             let mut sink = pending.accept().await?;
-///
-///             let msg = SubscriptionMessage::from_json(&"Response_A")?;
-///             sink.send(msg).await?;
-///
+///             let mut sink = pending.accept().await.map_err(|_| None)?;
+///             sink.send("Response_A".into()).await.map_err(|_| None)?;
 ///             Ok(())
 ///         }
 ///
 ///         // Send out two values on the subscription.
 ///         async fn sub(&self, pending: PendingSubscriptionSink) -> SubscriptionResult {
-///             let sink = pending.accept().await?;
+///             let sink = pending.accept().await.map_err(|_| None)?;
 ///
-///             let msg1 = SubscriptionMessage::from_json(&"Response_A")?;
-///             let msg2 = SubscriptionMessage::from_json(&"Response_B")?;
+///             let msg1 = SubscriptionMessage::from_json(&"Response_A").expect("Infallible");
```

Comment:
I don't get this yet; wouldn't `from_json` reject the invalid json "Response_A"?